vmdb/db/migrate/20130528153700_migrate_to_bcrypt_password.rb

```diff
@@ -6,6 +6,7 @@ class User < ActiveRecord::Base; end
 
   def up
     add_column   :users, :password_digest, :string
+    MiqPassword.add_legacy_key('v1_key')
```

Comment:
`add_legacy_key("file_not_found")` ran fine.
(of course it failed when trying to actually decrypt the file.)